models/load_dc_people.py

```diff
@@ -1,12 +1,34 @@
 """Load knowhys."""
 
+import re
+from typing import Any
+
 from bs4 import BeautifulSoup
 from langchain.schema.document import Document
 
 from models.load_utils import clean
 from models.load_utils import to_markdown
 
 
+def clean_text(markdown_content: str) -> Any:
+    """Remove unwanted texts from dc people content."""
+    # init return text
+    text_data = ""
+
+    # Search for the position of "abstract" (case insensitive)
+    text_match = re.search(r"\b[\d+](#t\d+).\b", markdown_content, re.IGNORECASE)
```

Comment:
I think this should be:

`r"\s\[\d+\]\(#t\d+\)\.\s"`

You have to escape [ ( and . to match them as characters because otherwise they have special meaning in regular expressions.

And the \b needs to be a \s because \b only matches a boundary between a word character and a non-word character, and the [ at the beginning and the . at the end are are non-word characters. \s instead matches space, tab, or newline, which is what you want.